dvc/config_schema.py

```diff
@@ -145,7 +145,7 @@ class RelPath(str):
                     "session_token": str,
                     Optional("listobjects", default=False): Bool,  # obsoleted
                     Optional("use_ssl", default=True): Bool,
-                    Optional("ssl_verify", default=True): Bool,
+                    Optional("ssl_verify", default=None): Any(Bool, str, None),
```

Comment:
I guess we could just

```suggestion
                    "ssl_verify": Any(Bool, str, None),
```

since they are optional by default.

Reply:
Yeah, that makes sense. And at that point I think I can get rid of the `None` so that it is `"ssl_verify": Any(Bool, str),`. I'm testing the change now and should be able to push soon.